hi-ml-cpath/src/health_cpath/datasets/panda_dataset.py

```diff
@@ -52,89 +38,3 @@ def __init__(self,
         self.dataset_df[self.IMAGE_COLUMN] = "train_images/" + slide_ids + ".tiff"
         self.dataset_df[self.MASK_COLUMN] = "train_label_masks/" + slide_ids + "_mask.tiff"
         self.validate_columns()
-
-
-# MONAI's convention is that dictionary transforms have a 'd' suffix in the class name
-class ReadImaged(MapTransform):
-    """Basic transform to read image files."""
-
-    def __init__(self, reader: ImageReader, keys: KeysCollection,
-                 allow_missing_keys: bool = False, **kwargs: Any) -> None:
-        super().__init__(keys, allow_missing_keys=allow_missing_keys)
-        self.reader = reader
-        self.kwargs = kwargs
-
-    def __call__(self, data: Dict) -> Dict:
-        for key in self.keys:
-            if key in data or not self.allow_missing_keys:
-                data[key] = self.reader.read(data[key], **self.kwargs)
-        return data
-
-
-class LoadPandaROId(MapTransform):
```

Comment:
Renamed to LoadMaskRoid so that we can use it for other cohorts without confusion